scripts/feature_tests/node_reuse/node_reuse.sh

```diff
@@ -12,4 +12,11 @@ export FROM_K8S_VERSION="v1.21.2"
 export KUBERNETES_VERSION=${FROM_K8S_VERSION}
 export UPGRADED_K8S_VERSION="v1.22.0"
 
+if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
+  export UPGRADED_IMAGE_NAME="UBUNTU_20.04_NODE_IMAGE_K8S_${UPGRADED_K8S_VERSION}.qcow2"
+  export UPGRADED_RAW_IMAGE_NAME="UBUNTU_20.04_NODE_IMAGE_K8S_${UPGRADED_K8S_VERSION}-raw.img"
+else
+  export UPGRADED_IMAGE_NAME="CENTOS_8_NODE_IMAGE_K8S_${UPGRADED_K8S_VERSION}.qcow2"
+  export UPGRADED_RAW_IMAGE_NAME="CENTOS_8_NODE_IMAGE_K8S_${UPGRADED_K8S_VERSION}-raw.img"
+fi
```

Comment:
Aren't we using same vars for upgrade? Shall we keep it somewhere in lib/images?

Reply:
No, I didn't, as we were testing only in ubuntu. I have created this var only for node reuse tests.